pipelines/test/no-docs.yaml

```diff
@@ -0,0 +1,105 @@
+name: no-docs
+
+needs:
+  packages:
+    - apk-tools
+    - man-db
+
+pipeline:
+  - name: primary package docs check
+    runs: |
+      # Get our primary package name
+      primary_pkg=$(basename ${{targets.contextdir}})
+
+      # Skip if this is a documentation package
+      case "$primary_pkg" in
+        *-doc)
+          echo "Skipping documentation package [$primary_pkg]"
+          echo "Remove the \"test/no-docs\" pipeline from documentation packages"
+          exit 0
+          ;;
+      esac
+
+      cd /
+      doc_violations=false
+      violation_count=0
+      has_manpages=false
+      has_infodir=false
+
+      # Start with default paths
+      default_paths="usr/share/man usr/local/share/man usr/man"
+      all_paths="$default_paths"
+
+      # Get additional paths from man --path
+      if command -v man >/dev/null 2>&1; then
```

Comment:
We have `man-db` listed in the `needs` section, so we should just error out if it doesn't exist for some reason. Then the rest of the code doesn't need to be in an `if` statement.

Also, if we're going to check that it exists, we might as well also check that it is executable (`test -x $(command -v man)`